backends/magma/ceed-magma-basis.c

```diff
@@ -253,19 +269,19 @@ int CeedBasisApplyNonTensor_Magma(CeedBasis basis, CeedInt nelem,
       for(int d=0; d<dim; d++) {
         if (d>0)
           beta = 1.0;
-        magma_dgemm(MagmaNoTrans, MagmaNoTrans,
-                    P, nelem*ncomp, Q,
-                    1.0, impl->dgrad + d*P*Q, P,
-                    du + d*nelem*ncomp*Q, Q,
-                    beta, dv, P);
+        magma_dgemm_nontensor(MagmaNoTrans, MagmaNoTrans,
+                              P, nelem*ncomp, Q,
+                              1.0, impl->dgrad + d*P*Q, P,
+                              du + d*nelem*ncomp*Q, Q,
+                              beta, dv, P, data->queue);
```

Comment:
There are a few places like this were the style conforms to `make style` but seems to be spread over a lot of lines?

Reply:
I'm not sure what is required here. Collapsing these lines can produce a very long line of code. 

From a BLAS perspective, the arguments in this call are organized in different lines because they semantically represent different pieces of information. The call begins with transpositions, then matrix sizes, information about the 'A' matrix, information about the 'B' matrix, and finally information about the 'C' matrix. It is easier to read that way for people who frequently use BLAS.

Reply:
If there is a reason for it, it's fine as is. I just noticed it and wasn't sure why it was arranged like that.